### ISSUE TITLE
[BUG] Bottom Aligned Snackbar on Android Tablet is concealed

### ISSUE THREAD

**Issue by mobilewares** [CommunityToolkit/Maui#1591]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Did you read the "Reporting a bug" section on Contributing file?

- [X] I have read the "Reporting a bug" section on Contributing file: https://github.com/CommunityToolkit/Maui/blob/main/CONTRIBUTING.md#reporting-a-bug

### Current Behavior

When a Community Toolkit Snackbar is shown on Android Tablet and aligned vertically to the bottom of screen, it is mostly concealed by the bottom action bar. Please see image below (blue Snackbar).

![image](https://github.com/CommunityToolkit/Maui/assets/4155775/5e1e3883-647b-4e9e-9c9b-a996c39e6418)

This was observed on Android 13 tablet using .NET8 MAUI (8.0.3) & Community Toolkit MAUI 7.0.0.

### Expected Behavior

The Snackbar should be displayed in the area assigned to the running App and not be concealed by elements outside the App.

### Steps To Reproduce

Use the Snackbar on Android 13 when the bottom action bar has been enabled. 
Note: This is enabled by default when using the default 'tablet_h_dpi_13_5in' emulator provided by the Android SDK.

### Link to public reproduction project repository

https://github.com/CommunityToolkit/Maui/tree/main/samples/CommunityToolkit.Maui.Sample

### Environment

```markdown
- .NET MAUI CommunityToolkit:7.0.0
- OS: Android 13
- .NET MAUI: 8.0.3
```

### Anything else?

I have not included my own repro because this should be easily reproducible simply running any Community Toolkit example app that uses the snackbar. I linked to the source code here to allow this to be submitted.

**Comment by vhugogarcia** [comment]
After reviewing this issue in-depth, I found the issue actually exists, I'm sharing my findings below:

- The issue occurs only on Android 13 and 14. I tested on Android 11 and 10 as well.
- The issue occurs only when the SnackBar is displayed from a modal page. This **does not** occur on normal content pages.
- The issue **does not** occurs on phones but only tablets.

### Workaround
- Add an **empty/blank anchor** element at the bottom of the page or in the position you would like to show the SnackBar on the page, and then display the SnackBar on that particular anchor _(I tested this solution and it worked OK on modal pages as well)_. `Snackbar.Make("Snackbar in a Modal Page",null, "OK",TimeSpan.FromSeconds(30), null, testLabel)`

### Source
This appears to be the source file that shows a snackbar: _\CommunityToolkit.Maui\Alerts\Snackbar\Snackbar.android.cs_

**Comment by ne0rrmatrix** [comment]
I tested android 13 and 14 with every type of snack bar using 3 different version of the emulator tablet. one 8 inch tablet, a 10 inch tablet, and a 13.5 inch tablet. I tested on a Samsung tab S8 plus with navigation bar turned on and off. None of them show the behavior now. They all draw the snackbar on screen and nothing is off screen. My suggestion is try latest dotnet 9.x with community toolkit 11.0. It should be working now.

**Comment by VladislavAntonyuk** [comment]
@ne0rrmatrix just to confirm, did you try from a modal page as @vhugogarcia suggested?

**Comment by ne0rrmatrix** [comment]
Yes I tried that. I will create a video demonstrating the expected behavior is present in sample app.

**Comment by ne0rrmatrix** [comment]
https://github.com/user-attachments/assets/781bf2e1-93d9-4d04-bcd4-3c90cfb02aa5

Sample video using Android Emulator table_h-dpi_13_5_in_-_api_33. I also tested with android 34.

**Comment by vhugogarcia** [comment]
Thanks @ne0rrmatrix for the help testing this using the latest versions.

We can safely now close this issue and if it appears again, we may re-open it or create a new one.
